doc/api/assert.md

```diff
@@ -110,6 +110,9 @@ parameter is an instance of an `Error` then it will be thrown instead of the
 added: v1.2.0
 changes:
   - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: zero is now compared using the [Object.is][] comparison.
```

Comment:
Nit: The fact that `Object.is()`  is used under the hood is probably not of interest to the end user compared to what the impact is for the end user. Like, if the big change is that `+0` and `-0` are no longer considered equal, maybe let's say that instead?

Reply:
Done